numojo/ndarray.mojo

```diff
@@ -421,14 +473,14 @@ struct NDArray[dtype: DType = DType.float32](Stringable):
     #     var index: Int = _get_index(indices, self.info.strides)
     #     self._arr.store[width=width](index, val)
 
-    fn num_elements(self) -> Int:
```

Comment:
Keep this for compatibility so I can build a trait to allow functions to use either our ndarray or the standard tensor. Or have an alias to size function called num_elements.

Reply:
It is added back in the new commit.

Reply:
the commit 7253e52b solved this.